src/components/LPList/index.js

```diff
@@ -122,7 +122,7 @@ function LPList({ lps, disbaleLinks, maxItems = 10 }) {
         <DataText>
           <CustomLink area="pool" to={'/pool/' + lp.poolAddress}>
             <RowFixed>
-              {!below600 && <TokenLogo address={lp.token} size={16} margin={true} />}
+              {!below600 && <TokenLogo address={lp.token} size={'16px'}/>}
```

Comment:
why change style here?

Reply:
This margin parameter does not exist and make console print error